app/components/EventItem/styles.css

```diff
@@ -1,11 +1,18 @@
 .eventItem {
   min-height: 50px;
+  padding: var(--spacing-xs);
   padding-left: var(--spacing-md);
   border-left: 4px solid transparent;
   margin-bottom: var(--spacing-md);
   display: flex;
   justify-content: space-between;
   line-height: 1.3;
+  transition: var(--easing-fast);
+}
+
+.eventItem:hover {
+  background-color: var(--additive-background);
+  border-radius: var(--border-radius-sm);
```

Comment:
Joblistings use `--border-radius-lg`. I would love for this to be consistant with that

Reply:
I also removed the radius from the left side
It is necessary when the bordrer radius is that big